plugins/aps/src/main/assets/OpenAPSSMBDynamicISF/determine-basal.js

```diff
@@ -227,14 +227,24 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         //sensitivityRatio = 2/(2+(target_bg-normalTarget)/40);
         var c = halfBasalTarget - normalTarget;
         sensitivityRatio = c/(c+target_bg-normalTarget);
-        // limit sensitivityRatio to profile.autosens_max (1.2x by default)
-        sensitivityRatio = Math.min(sensitivityRatio, profile.autosens_max);
-        sensitivityRatio = round(sensitivityRatio,2);
```

Comment:
This fix destroys the possibility of reactivating the exercise mode of high/low temptarget raises/lowers sensitivity. This is because it is "only" limited in one direction, "downwards" (autosens_max). As it was, it was correct.

Reply:
Yeah, seems I had some other assumptions wrong as well. Will remove this PR. and back to talk to some ppl about this.